feature/agenda/presentation/src/main/java/com/sergius/agenda/presentation/agendaoverview/AgendaScreenRoot.kt

```diff
@@ -139,6 +155,132 @@ private fun AgendaScreen(
                         .padding(vertical = 4.dp, horizontal = 12.dp),
                     textAlign = TextAlign.Left
                 )
+
+                LazyColumn {
+                    items(state.items) {
+                        AgendaItemUi(
+                            onAction = onAction,
+                            item = it,
+                            showMoreActions = state.showMoreActions
+                        )
+                    }
+                }
+            }
+        }
+    }
+}
+
+@Composable
+private fun AgendaItemUi(
+    onAction: (AgendaAction) -> Unit,
+    item: AgendaItemUiData,
+    showMoreActions: Boolean
```

Comment:
Do we actually want a shared flag that controls the dropdown for all items? The downside is that one toggle can open/close menus across the entire list, leading to confusing UX and hard-to-reason state.
You can try some state management like this: 
```kotlin
// In AgendaState.kt
data class AgendaState(
    // ... other properties
    val expandedItemId: String? = null  // Track which specific item is expanded
)

// In AgendaAction.kt
data class OnToggleMoreActions(val itemId: String) : AgendaAction

// In AgendaItemUi composable
showMoreActions = state.expandedItemId == item.id
```